### DISCUSSION TITLE
关于 extendsMarkdown 中的 markdown-it 实例

### DISCUSSION THREAD

**Discussion by waycowei** [vuepress/core#400] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/8675871/130409383-c0f0f10c-82ff-44bc-9673-8cb7a429d78d.png)
因为插件用到了这个实例, 确认一下, Vuepress1中的这个 render 的 override 是不再支持了是吧?

**Comment by meteorlxy** [comment, marked as answer]
v1 这种 hack 的方式本身并不是 markdown-it 的正常用法。

v2 将使用 markdown-it 的正常使用方式，通过 `env` 变量来传入和返回额外的信息。
